packages/hyperion-autologging/src/ALSurface.ts

```diff
@@ -290,7 +290,6 @@ export function init(options: InitOptions): ALSurfaceRenderers {
         wrapperElementType,
         {
           [SURFACE_WRAPPER_ATTRIBUTE_NAME]: "1",
-          style: { display: 'contents' },
```

Comment:
So I assume as we discussed offline, you're going to add the `wrapperStyle` as a field of the capabilities, correct?